packages/web-app-files/src/components/Modals/SetLinkPasswordModal.vue

```diff
@@ -65,7 +65,9 @@ export default defineComponent({
       } catch (e) {
         // Human-readable error message is provided, for example when password is on banned list
         if (e.response?.status === 400) {
-          errorMessage.value = $gettext(e.response.data.error.message)
+          const errorMsg = e.response.data.error.message
+          const capitalizedErrorMsg = errorMsg.charAt(0).toUpperCase() + errorMsg.slice(1)
```

Comment:
`lodash-es` has an `upperFirst` helper that could be used here.

Reply:
Oh nice, I didn't knew, thanks!